mscore/propertymenu.cpp

```diff
@@ -588,5 +557,43 @@ void ScoreView::editTremoloBarProperties(TremoloBar* tb)
                   score()->undo(new ChangeTremoloBar(static_cast<TremoloBar*>(b), bp.points()));
             }
       }
+
+//---------------------------------------------------------
+//   selectInstrument
+//---------------------------------------------------------
+
+void Ms::ScoreView::selectInstrument(InstrumentChange* ic)
```

Comment:
The function name `selectInstrument()` is not very clear, if you don't know it's related to instrument change. Maybe something like `selectNewInstrument()` or simply `changeInstrument()` is better?

Reply:
I think it should match the title of the dialog it opens, but that also could be "Change Instrument."

Reply:
Actually, SelectInstrument is also the internal name of the object. I'm inclined to keep it that way.